packages/cli/cloud/src/services/strapi-info-save.ts

```diff
@@ -1,22 +1,36 @@
 import fse from 'fs-extra';
 import path from 'path';
-import type { ProjectInfos } from './cli-api';
+import type { ProjectInfo } from './cli-api';
 
 export const LOCAL_SAVE_FILENAME = '.strapi-cloud.json';
 
 export type LocalSave = {
-  project?: Omit<ProjectInfos, 'id'>;
+  project?: Omit<ProjectInfo, 'id'>;
 };
 
-export async function save(data: LocalSave, { directoryPath }: { directoryPath?: string } = {}) {
+export async function addEnvironment(
+  targetEnvironment: string,
+  { directoryPath }: { directoryPath?: string } = {}
+) {
   const alreadyInFileData = await retrieve({ directoryPath });
-  const storedData = { ...alreadyInFileData, ...data };
+  const storedData = {
+    ...alreadyInFileData,
+    project: {
+      ...alreadyInFileData.project,
+      targetEnvironment,
+    },
+  };
   const pathToFile = path.join(directoryPath || process.cwd(), LOCAL_SAVE_FILENAME);
-  // Ensure the directory exists
   await fse.ensureDir(path.dirname(pathToFile));
```

Comment:
I believe we should re-use the `save` function? (benefits: typings check on the data we write in the file)

Reply:
Yes, we can do that.